probe-rs-tools/src/bin/probe-rs/util/cli.rs

```diff
@@ -525,12 +525,11 @@ async fn print_monitor_event(
 
             processor.process(&bytes);
         }
-        MonitorEvent::SemihostingOutput(SemihostingOutput::StdOut(str)) => {
-            print!("{}", str)
-        }
-        MonitorEvent::SemihostingOutput(SemihostingOutput::StdErr(str)) => {
-            eprint!("{}", str)
-        }
+        MonitorEvent::SemihostingOutput { stream, data } => match stream.as_str() {
+            "stdout" => print!("{data}"),
+            "stderr" => eprint!("{data}"),
+            _ => {}
```

Comment:
What is this intended for in the future? :)

Reply:
I was thinking about defmt (via defmt-semihosting)